### ISSUE TITLE
Document packaging

### ISSUE THREAD

**Issue by TheEvilSkeleton** [Lea-fish/Leafish#123]
Since we now have a desktop entry and a metainfo file for Linux users, it'd be best to document on packaging on Linux.

Just an FYI, the following line will need to be removed sooner or later:

https://github.com/Lea-fish/Leafish/blob/08a98f0135255b40e37037fd965afb0874d6bbdb/packaging/leafish.metainfo.xml#L25

**Comment by PureTryOut** [comment]
Wait, why would documentation be needed? I assume distro packagers already know how to package a Rust-based package, and there is nothing special here.

**Comment by TheEvilSkeleton** [comment]
I thought I was already clear, sorry for that.

The cargo part is the easiest part, but everything lying under the `packaging` directory and the icons are the "tricky" parts, because the packager will have to add the files in the needed locations.

Not everyone likes using cargo, myself included. I prefer using native package managers to install packages, whether they use cargo or not, that's another story.

Just my personal opinion. If you are against it, then I'm completely okay with closing the issue.

**Comment by PureTryOut** [comment]
I'm not necessarily _against_ it, I just don't see much of a reason for it. I prefer native distro packages as well, but they are still made using cargo. Besides building the binary distributions will have to install the desktop file and icons, but to a distro packager that is all quite obvious and they know where it all should go (I can know, I am one (for Alpine Linux)).

**Comment by TheEvilSkeleton** [comment]
True. I'll see if someone objects. I don't mind closing this issue if deemed unecessary.
